e2e/teams.spec.ts

```diff
@@ -78,16 +82,18 @@ test.describe('Teams', () => {
           await expect(teamNameInput).toHaveValue(TEAM_NAME)
 
           await userPage.getByRole('button', { name: 'OK' }).click()
+
+          await testAuth.step('user should get teamId', async () => {
+            await userPage.waitForURL(`/${teamHashBang}:**`)
+            teamId = getTeamIdFromPathname(userPage.url())
+          })
```

Comment:
Get the teamId so that the team can be cleaned up in the afterAll if the test fails before the team is deleted via the UI